File: ARCADE/The Core/Level 2_5: arithmeticExpression.cs
    using System;
     
    public class Test
    {
    	public static bool arithmeticExpression(int A, int B, int C){
            if (A + B == C || A - B == C || A * B == C || (A % B == 0 && A / B == C)) return true;
            return false;
        }
    	public static void Main()
    	{
    		int A = 2, B = 3, C = 5;
    		Console.WriteLine(arithmeticExpression(A, B, C));
    		int a = 8, b = 3, c = 2;
    		Console.WriteLine(arithmeticExpression(a, b, c));
    	}
    }

File: ARCADE/The Core/Level 2_5: arithmeticExpression.pas
program arithmeticExpression;

function arithmeticExpression(a: integer; b: integer; c: integer): boolean;
begin
    if((A + B = C) or (A - B = C) or (A * B = C) or ((A mod B = 0) and (A div B = C))) then arithmeticExpression := true
    else arithmeticExpression := false;
end;

var a, b, c : integer;
begin
	a := 2; b := 3; c := 5;
	writeln(arithmeticExpression(a, b, c));
	a := 8; b := 2; c := 4;
	writeln(arithmeticExpression(a, b, c));
	a := 8; b := 3; c := 2;
	writeln(arithmeticExpression(a, b, c));
end.
